ddk-configuration/checkstyle/avaloq-test.xml

```diff
@@ -1,156 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would not remove this file even if the settings were the same like the normal ones. We often had cases where we wanted to have differences between normal and test code. You have an example below that should be configured in here.